components/reusable-components/turnstile-captcha/CaptchaWidget.tsx

```diff
@@ -1,105 +1,28 @@
-import React, { useEffect, useRef } from 'react';
+import React from 'react';
+import Turnstile from 'react-turnstile';
 
-interface CaptchaProps {
-  onSuccess?: () => void;
-  onError?: () => void;
-  onExpired?: () => void;
-  siteKey?: string;
-  cData?: string;
-  theme?: 'auto' | 'light' | 'dark';
-  language?: string;
-  tabIndex?: number;
-}
+const TurnstileWidget: React.FC = () => {
```

Comment:
Unnecessary rename, you shouldn't change the function name.

Reply:
Additionally: you've modified this entire class but you do not use it anywhere. 
In all other files, you're directly importing `react-turnstile` component, which makes this component obsolete.

Reply:
Hint: Delete obsolete code.